guides/common/modules/ref_prerequisites-for-using-ipxe.adoc

```diff
@@ -0,0 +1,30 @@
+[id="prerequisites-for-using-ipxe_{context}"]
+= Prerequisites for using iPXE
+
+You can use iPXE to boot virtual machines on hypervisors that use iPXE as primary firmware.
+Furthermore, you can configure PXELinux to chainboot iPXE in BIOS mode and boot by using the HTTP protocol if you have the following restrictions that prevent you from using PXE:
+
+* A network with unmanaged DHCP servers.
+* A PXE service that is unreachable because of, for example, a firewall restriction.
+* A TFTP UDP-based protocol that is unreliable because of, for example, a low-bandwidth network.
+
+For booting systems in UEFI mode by using HTTP, follow xref:Creating_Hosts_with_UEFI_HTTP_Boot_Provisioning_{context}[] instead.
+
+ifndef::satellite[]
```

Comment:
@jloeser does this make sense for orcharhino?

Reply:
Let's stick to Satellite version with orcharhino.

Should basically work on BIOS and EFI firmware platforms.